lib/components/narrative/realtime-annotation.js

```diff
@@ -25,28 +27,35 @@ export default class RealtimeAnnotation extends Component {
     const innerContent = <div className='realtime-alert'>
       <div className='content'>
         <h3>
-          <i className='fa fa-exclamation-circle' /> Service update
+          <i className='fa fa-exclamation-circle' />
+          {' '}
```

Comment:
I think adding whitespace is a no-no when using internationalization. If this is for styling purposes to add padding or something, the styling should be changed instead.